### ISSUE TITLE
Finalize design for Chicago report 

### ISSUE THREAD

**Issue by LauraReynal** [MozillaFoundation/Design#71]
Team: @lovegushwa and I
Decision/review: @cassiemc 

Milestone: Friday 14th 

**Goal**: we need to make some changes to the editorial design of the research report document and to finish the design of the different graphics and frameworks.

To do
- [x] Refresh the editorial design of the inDesin template in order to answer various communication needs. Design a few new Master Pages with different grids for this purpose. 
* pull out boxes
* big quotes and small quotes
* Footnotes (tedious and not in the grid today)
* Side bar for recommended readings 
 
- [x] Finalize the web journey - WIP @lovegushwa 
- [x] Finalize the threshold diagram - WIP - what needs to be done. Is design finished ?

**Comment by LauraReynal** [comment]
@lovegushwa to make things easier and give you a glimpse of the different editorial needs of the report itself, I pinged you in the google doc in various places.

**Comment by LauraReynal** [comment]
@lovegushwa here is the drive folder where you can find all the Chicago assets. Feel free to version your copies of the indesign documents into this folder. 

https://drive.google.com/open?id=0B0X05aIHuAbed2xfX0Z5c09KaTg

I will start porting the content into the document for you to be able to work on it. Thansk !

**Comment by LauraReynal** [comment]
@lovegushwa I can't find the document CHICAGO_X copy-Jordan-B anywhere 

We are working on Research Reports Assets > Chicago in google drive. 

Did you upload it there?

**Comment by LauraReynal** [comment]
It seems like the Chicago X folder itself has disapeared from the drive folder.

**Comment by lovegushwa** [comment]
@LauraReynal @cassiemc  ... Just updating the universe. The files were in google doc limbo. Laura and I are collaborating (I'm working with her to finish tables and charts she needs and cleaning up the new report template.) We have a daily standup at 10am(PST) in Laura's room.

**Comment by cassiemc** [comment]
Ok cool :)

**Comment by LauraReynal** [comment]
@lovegushwa I put lots of content in the inDesign document for you to be able to work on the master pages and the grid. 

Document name is Chicago X- Copy 11-08-15

There should be every style of content in there: 
- Basic text,
- Headings 1, 2, 3 
- Normal quotes 
- BIG quotes 
- pull out boxes for anything related to side content (legends, side-stories, recommendations etc)
- footnotes 
- full page pictures (not for print but for the PDF -- I know they look bad on print )
- the correct copy of the skills and constraints 

PS: I can't put the whole document inside, it would take me way too much time and also it means that I would need to do it twice since I dont have a copy / content review yet.

**Comment by lovegushwa** [comment]
Excellent @LauraReynal !
I will work on these style elements this evening. Goal for tomorrow is a solid working template.

**Comment by lovegushwa** [comment]
@LauraReynal Updated the file. I rebuilt all the type styles, the grid and master pages.  I still need to crack skills and constraints, lets go over the document together tomorrow during standup. Please continue to add content.

**Comment by LauraReynal** [comment]
Jordan, you won't believe me: The file you worked on was not the correct one. Another Drive mess up. It did not save into the folder properly I guess, or it disapeared again. :crying_cat_face: 

Let's clean this mess when we speak tonight. I can still use most of what you did, which is awsooome !

**Comment by LauraReynal** [comment]
If you wake up before our stand / have time here is the file I intended to give you yesterday https://drive.google.com/open?id=0B0X05aIHuAbeWjF2RnAtV0JqdEk

**Comment by LauraReynal** [comment]
Note for our stand-up tonight: We also need a page with multiple images. I used to make them myself in the other reports, and im not sure I have proportions correct. 
Examples: 

![screen shot 2015-08-12 at 10 55 46](https://cloud.githubusercontent.com/assets/8261945/9221376/c6d9624e-40e1-11e5-9838-1aa4a9e6e812.png)
![screen shot 2015-08-12 at 10 55 54](https://cloud.githubusercontent.com/assets/8261945/9221377/c6da36a6-40e1-11e5-996d-4a919399fbd2.png)
![screen shot 2015-08-12 at 11 02 26](https://cloud.githubusercontent.com/assets/8261945/9221378/c6de23a6-40e1-11e5-817c-7b0915c2f8ae.png)
![screen shot 2015-08-12 at 11 02 31](https://cloud.githubusercontent.com/assets/8261945/9221379/c6dfc012-40e1-11e5-8339-ceaf885f93e1.png)

**Comment by LauraReynal** [comment]
For the personas page, I'm thinking of something like this. 
Will put content for you in the file.
![img_20150812_155812](https://cloud.githubusercontent.com/assets/8261945/9227592/52680980-410c-11e5-8cc5-3d7c4db18d79.jpg)

**Comment by lovegushwa** [comment]
@LauraReynal I made good progress today. The updated file is on the drive for you to use. Let's brainstorm the grid tomorrow at standup.

**Comment by LauraReynal** [comment]
@lovegushwa thanks :smile: :tulip: 
A few notes for our standup tonight : 
- we lost the 2 boxes page 3 which described Mofo learning producst and programs
- I really love what you did for the skills and constraints !! Super clean and easy to read :yellow_heart: The only things is I would not put color around "no use" and "creation" - it makes me only see them, but they arent the most important 
- for the constraints, I think we should stick to one wording: either constraints or blocker. We can discuss. 
- love what you did for page 22-23 
- For the page 42 -43 Is there a way to put 3 pictures ? I even initally wanted 4. I think this is important. 
Thanks for the huge progress :+1:

**Comment by LauraReynal** [comment]
- I made a few changes to the threshold diagrams. I made 2 new spreads inside the doc. I wanted to try something out, but im not attached to the photos. However, if we can work out the quotes it would be awsome. Let's discuss tonight :) 
Aslo added the educators copy - which we did not have yet

**Comment by LauraReynal** [comment]
@lovegushwa thanks for the design pass yesterday -- Love it :+1: 

Is it done on your side or was there other things you wanted to do  ? 

I had to replace the glyphs you used for the recommendations because you used a special font that I did not have on my computer. So I used a normal arrow from Fira Sans.
Let me know if it's okay.

**Comment by cassiemc** [comment]
I am going to help finalize the design here, and will do it in the next heartbeat provided the more thorough copy review gets done in time. Updating the milestone.

**Comment by cassiemc** [comment]
As I was going through the file tonight I started to realize there were lots of pages that were wonky, probably due to my older version of indesign. I finally got my service now request for creative cloud this week and  am installing now. I have made lots of updates to the photographs, colors, paragraph styles and the grid on master pages – including updating the document size so it prints better – really important first steps. 

But rather than spend my whole weekend on this, I am going to update the milestone. So sorry @LauraReynal and @kevinzawacki -- I am blocking off all but an hour next Tuesday to finish this.
